### PR TITLE
Serve static content

### PR DESCRIPTION
**Pull request by Sybrand**
- move api from / to ./api
- serve up static content from /
- serve up /config.js as template

### CODE REVIEW

File: api/app/frontend.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+
+templates = Jinja2Templates(directory='static')
+
+
+async def get_config(request: Request):
+    """ Apply jinja template response to config.js
+    """
+    return templates.TemplateResponse(
+        "config.js",
```
Comment (Kyubinhan) [inline api/app/frontend.py:36]: I wonder if you need to create `config.js` file under `/static`

Comment (Sybrand) [reply api/app/frontend.py:36]: I suspect I might have to do that just to get the code coverage good.

**Review by Kyubinhan — COMMENTED** [review]
Are you intended to merge this pr? if so, we would have to make changes on FE for url changes

**Comment by Sybrand** [comment]
> Are you intended to merge this pr? if so, we would have to make changes on FE for url changes

Yep - there's a PR for the front end, and we'd have to change the env variables when deploying.

**Review by Kyubinhan — APPROVED** [review]

**Review by andrea-williams — APPROVED** [review]

**Pull request merged**